exercises/triangle/example.tt

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'triangle'
+
+# Test data version:
+# <%= sha1 %>
+class TriangleTest < Minitest::Test<% test_cases.each do |test_case| %>
+  def <%= test_case.name %><% if test_case.skipped? %>
+    skip<% end %>
+    triangle = Triangle.new(<%= test_case.sides %>)<% if test_case.expected%>
```

Comment:
I would consider moving some of this logic into the `workload` method once it is defined